lib/rubocop/cop/internal_affairs/node_matcher_directive.rb

```diff
@@ -103,6 +103,7 @@ def too_many_directives(node)
 
         def process_directive(node, actual_name, directive)
           return unless (offense_type = directive_offense_type(directive, actual_name))
+          return unless node.arguments.count == 2
```

Comment:
I'd add a comment explaining this, as it's quite cryptic without one.

Reply:
I didn't even add it add the proper place, oops. Right after `on_send` it doesn't need one I think but let me know if you still think otherwise.

Reply:
The intent of the modified code is clearly readable after this comment.